### PR TITLE
Added FavoriteView and changed to double quote 

### PR DESCRIPTION
**Pull request by lbc1013**
@winstonthep  @leonardodeutsch @Heine574 

Ticket : https://trello.com/c/M0pqYhD9/26-favorite-view-component,
https://trello.com/c/k4WwS0rq/27-delete-the-recommend-recipes-scroll-option

- Added the FavoriteView.
- Deleted the scroll option for recipes list.
- Added a function that the user can go to exploreView, when they click the explore button at the homePage.
- Changed to double quote for html related and camelCase for css related in jsx.

### CODE REVIEW

File: src/components/FavoriteView.jsx [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
+        <RecipeTile/>
+        <RecipeTile/>
+        <RecipeTile/>
+        <RecipeTile/>
+        <RecipeTile/>
+        <RecipeTile/>
+        <RecipeTile/>
+        <RecipeTile/>
+        <RecipeTile/>
+        <RecipeTile/>
```
Comment (Heine574) [inline src/components/FavoriteView.jsx:20]: I assume these are just placeholders until we actually load user information?

File: src/App.jsx [hunk 2]
```diff
@@ -55,6 +57,9 @@ class App extends React.Component {
   handleClose() {
     this.setState({showLogin: false});
   }
+  toHandleExploreView () {
```
Comment (Heine574) [inline src/App.jsx:60]: This seems redundant... could you not just use captureNavigation for this instead of creating a new function to pass down?

**Review by Heine574 — CHANGES_REQUESTED** [review]

**Review by Heine574 — APPROVED** [review]

File: src/components/FavoriteView.jsx (hunk 1 above)
Comment (lbc1013) [reply src/components/FavoriteView.jsx:20]: yes it's just placeholders before we load the actual data.

File: src/App.jsx (hunk 2 above)
Comment (lbc1013) [reply src/App.jsx]: adjusted. thanks!

**Pull request merged**